relnotes/makegroup.feature.md

```diff
@@ -0,0 +1,10 @@
+### Add helper method to create GroupRequest
+
+* `dlsproto.client.legacy.internal.helper.GroupRequest`
+
+  Previously, to instantiate the GroupRequest, the user had to pass the request
+  struct as a template parameter, since the constructor wouldn't deduce it.
+  Now the helper method `makeGroupRequest` is added which instantiates and
+  returns the right `GroupRequest` instance without the need for the user to
+  specify the template parameters.
+
```

Comment:
Is it worth mentioning the specific situation where this is useful? i.e. using DMD 2.071.*+.

Reply:
It's not just that. The code looks nicer. It's better in every way.

Reply:
I've updated it slightly, though, just to have a reference for poor souls that are using newer compiler.